yarn.lock

```diff
@@ -14826,19 +14840,20 @@ shallowequal@^1.1.0:
   version "1.1.0"
   resolved "https://registry.yarnpkg.com/shallowequal/-/shallowequal-1.1.0.tgz#188d521de95b9087404fd4dcb68b13df0ae4e7f8"
 
-sharp@0.26.2:
-  version "0.26.2"
-  resolved "https://registry.yarnpkg.com/sharp/-/sharp-0.26.2.tgz#3d5777d246ae32890afe82a783c1cbb98456a88c"
-  integrity sha512-bGBPCxRAvdK9bX5HokqEYma4j/Q5+w8Nrmb2/sfgQCLEUx/HblcpmOfp59obL3+knIKnOhyKmDb4tEOhvFlp6Q==
+sharp@0.26.3:
+  version "0.26.3"
+  resolved "https://registry.yarnpkg.com/sharp/-/sharp-0.26.3.tgz#9de8577a986b22538e6e12ced1f7e8a53f9728de"
+  integrity sha512-NdEJ9S6AMr8Px0zgtFo1TJjMK/ROMU92MkDtYn2BBrDjIx3YfH9TUyGdzPC+I/L619GeYQc690Vbaxc5FPCCWg==
   dependencies:
-    color "^3.1.2"
+    array-flatten "^3.0.0"
```

Comment:
Oh, god. Every new release - more and more dependencies 👎

Reply:
You're right, that is regrettable. Luckily `sharp` is optional, although that is broken atm.

Reply:
Not really optional. It's always installed if environment is supported. It was decided to make it mandatory to easily switch to alternative. Though there is not any. Perhaps optimisation should be left on user initially without introducing bloated dep.

Reply:
You should be able to omit it when using `yarn install --ignore-optional`. Or am I missing something? In any case, optional dependencies is not the most commonly used feature of yarn/npm to say the least.

Reply:
Well, yeah. Though flags are not really handy and some other optionalDependencies can be necessary.